.changeset/fifty-berries-learn.md

```diff
@@ -0,0 +1,37 @@
+---
+'@backstage/plugin-catalog': patch
```

Comment:
```suggestion
'@backstage/plugin-catalog': minor
```

Reply:
Done please checkout newer commit, I was wondering between minor and patch earlier :)